automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/BillingFixtures.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.dsde.firecloud.fixture
+
+import org.broadinstitute.dsde.firecloud.api.Rawls
+import org.broadinstitute.dsde.firecloud.auth.AuthToken
+import org.broadinstitute.dsde.firecloud.config.{Config, UserPool}
+import org.broadinstitute.dsde.firecloud.test.{CleanUp, WebBrowserSpec}
+import org.scalatest.Suite
+
+trait BillingFixtures extends CleanUp { self: WebBrowserSpec with Suite =>
+
+  def withBillingProject[T](billingProjectPrefix: String)(f: String => T)(implicit authToken: AuthToken): T = {
```

Comment:
Wonder if this could use workbench-libs `GoogleProject` instead of `String`?

Reply:
Yes - Swat has several Yaksworth of this sort of thing